src/main/java/cms/gov/madie/measure/services/VersionService.java

```diff
@@ -110,5 +120,14 @@ protected void getNextVersion(Measure measure, String versionType) {
     } else if (VERSION_TYPE_PATCH.equalsIgnoreCase(versionType)) {
       measure.getVersion().setRevisionNumber(measure.getVersion().getRevisionNumber() + 1);
     }
+    return formatVersion(measure);
+  }
+
+  protected String formatVersion(Measure measure) {
+    return String.valueOf(measure.getVersion().getMajor())
+        + "."
+        + String.valueOf(measure.getVersion().getMinor())
+        + "."
+        + StringUtils.leftPad(String.valueOf(measure.getVersion().getRevisionNumber()), 3, '0');
```

Comment:
Version.toString() already has similar formatting. Can we reuse ?

Reply:
@RohitKandimalla Thanks so much for the tip and info. I've removed the unnecessary format code.